util/os-resources.py

```diff
@@ -0,0 +1,84 @@
+# !/usr/bin/python3
```

Comment:
Minor: what about documenting an example of usage? It could be in REAMDE, in a comment on top of this file or, even better, included in the script helper

Reply:
added, thanks for the suggestion!